backend/controllers/appointmentConfirmDeletion.ts

```diff
@@ -53,8 +55,23 @@ export default ({ whereaboutsApi, appointmentDetailsService }) => {
 
     if (isRecurring === 'true') return res.redirect(`/appointment-details/${id}/delete-recurring-bookings`)
 
+    const appointmentDetails = await whereaboutsApi.getAppointment(res.locals, id)
+
     try {
-      await whereaboutsApi.deleteAppointment(res.locals, id)
+      let videoLinkBooking
+
+      const systemContext = await getClientCredentialsTokens(res.locals.user.username)
+
+      if (config.apis.bookAVideoLinkApi.enabled && appointmentDetails.appointment.appointmentTypeCode === 'VLB') {
+        videoLinkBooking = await videoLinkBookingService.getVideoLinkBookingFromAppointmentId(systemContext, id)
+      }
```

Comment:
Switch is here!

Reply:
Does it still require the user to have the ROLE_DELETE_APPOINTMENTS to remove a `new` video link booking?